src/prelaunch_data_generator/cl_validator_keystores/cl_validator_keystore_generator.star

```diff
@@ -72,12 +74,16 @@ def generate_cl_validator_keystores(
 
 	# Store outputs into files artifacts
 	keystore_files = []
+	running_total_validator_count = 0
 	for idx, participant in enumerate(participants):
 		output_dirpath = all_output_dirpaths[idx]
-
+		if participant.validator_count == 0:
+			keystore_files.append("dummy")
```

Comment:
lets take the dummy out as a constant, also shouldn't this be `None` as we check if node_keystore_files is none downstream